tests/unit/igraph_edge_betweenness.out

```diff
@@ -26,3 +26,14 @@ Testing bug 950, tolerances
 ( 0 0 2.33333 2.33333 2.33333 0 2.33333 2.33333 2.33333 2.33333 2.33333 2.33333 0 0 0 )
 
 Testing bug 1050, cutoff values
+
+Testing eids parameter edge selection
+Full edge betweenness: ( 1 1 1 1 1 1 )
```

Comment:
If all betweenness values are the same, then it's impossible to test that the subset is taken correctly. Use an example where some values are different.